.gitignore

```diff
@@ -1 +1,3 @@
-*build
\ No newline at end of file
+*build
```

Comment:
The build instructions for beman.exemplar has builds going in the `build` directory so it makes sense it would be part of `.gitignore`. Is the `*` necessary?

Also, should this be `/build`?

Reply:
Whoops, you're right